src/couch_expiring_cache/README.md

```diff
@@ -0,0 +1,20 @@
+# Couch Expiring Cache
+
+Key value cache with entries which are automatically deleted after
+they expire.
+
+This is a description of some of the modules:
+
+* `couch_expiring_cache`: The API module, it contains functions for
+  inserting and looking up cache entries, which are simply
+  pass-throughs to `couch_expiring_cache_fdb`.
+
+* `couch_expiring_cache_fdb`: The module which interacts with FDB, in
+  addition to insertion and lookup functions, it also contains a
+  function to clear an expired range.
+
+* `couch_expiring_cache_server`: A gen_server, this module
+  periodically removes expired cache entries using configurable
+  parameters for period, jitter, and batch size. It also fetches the
+  layer prefix on startup, and caches it in an ets table for
+  performance.
```

Comment:
Maybe add the description from the PR comment about the general need for this, how it works, the data model. A link to the FDB forum.

Reply:
Good call! I pushed a commit to improve the README.